src/Pos/Generator/Block/Param.hs

```diff
@@ -49,6 +49,8 @@ data BlockGenParams = BlockGenParams
     -- 'toPublic'.
     , _bgpBlockCount :: !BlockCount
     -- ^ Number of blocks to generate.
+    , _bgpInplaceDB  :: !Bool
+    -- ^ Whether to rewrite existing DB.
```

Comment:
I would say `extend existing DB` rather than `rewrite`, but it's minor and I may be wrong.